app/adapters/alma/adapter.rb

```diff
@@ -0,0 +1,36 @@
+module Alma
+  class Adapter
+
+    def initialize(connection:)
```

Comment:
Are you planning to pass in a different connection? If not I don't think we need this keyword param and I don't think we even need an initializer method.

Reply:
yes this was left from the initial work I was doing